feat(docs): Add docs for Opsgenie config and install (#7651)
getsentry/sentry-docs: src/docs/product/integrations/notification-incidents/opsgenie/index.mdx

---
title: "Opsgenie Integration"
sidebar_order: 1
description: "Learn more about Sentry's Opsgenie integration, which allows you to send issue and metric alerts to Opsgenie teams of your choice."
---

The Opsgenie integration allows you to connect your Sentry organization with one or more Opsgenie accounts. Once your accounts have been connected, you'll be able to configure issue and metric alerts in Sentry to be sent to the Opsgenie teams of your choice.

## Install

<Note>

You'll need Sentry owner, manager, or admin-level permissions to be able to install this integration.

</Note>

1. To get started, go to **Settings > Integrations > Opsgenie (Integration)** and click "Add Installation".

   ![Install Opsgenie integration](opsgenie-overview.png)

> 2. Select your base Opsgenie URL, type in your Opsgenie account name, and optionally add an integration key. Then, click 'Submit'. You'll be directed to the configuration page for your connected Opsgenie integration.

Reviewer: ```suggestion
2. Select your base Opsgenie URL and type in your Opsgenie account name. If you like, you can also add your integration key at this time. After you click 'Submit', you'll be directed to the configuration page for your connected Opsgenie integration.
```

   ![Opsgenie installation form](opsgenie-form.png)

## Configure

The Opsgenie integration should now be installed. Go to the configuration page to manually add and update integration keys. (Keys have to be associated with a 'Sentry' integration in Opsgenie.)

> If you added an integration key during installation, it will appear on the configuration page with the label 'my-first-key'. You may change this label if you wish.

Reviewer: ```suggestion
If you added an integration key during installation, it'll appear on the configuration page with the label 'my-first-key'. You can change this label at any time.
```

![Opsgenie configuration page](opsgenie-config.png)

### Set Up Alerts

> You can set up a new alert rule via the Opsgenie integration configuration page clicking on the **Add Alert Rule** button in the **Project Configuration** section, or by going to the **Alerts** page in Sentry.

Reviewer: ```suggestion
You can set up a new alert rule via the Opsgenie integration configuration page by clicking on the **Add Alert Rule** button in the **Project Configuration** section, or by going to the **Alerts** page in Sentry.
```

> In [issue alerts](/product/alerts/alert-types/#issue-alerts), select "Send an Opsgenie notification" in the actions dropdown and then select your account and team:

Reviewer: ```suggestion
To create an [issue alert](/product/alerts/alert-types/#issue-alerts) in Sentry, go to the **Alerts** page and click "Create Alert". After you've decided what you want to be alerted about, you'll be able to choose to "Send an Opsgenie notification" from the "Add action" dropdown. You can then choose your account and team:
```

![Opsgenie issue alert rule action](opsgenie-issue-alert-rule.png)

> In [metric alerts](/product/alerts/alert-types/#metric-alerts), select the Opsgenie option in the dropdown that corresponds to your account and the select your team:

Reviewer: ```suggestion
If you'd rather be alerted when a specific metric crosses a threshold, you can set a [metric alert](/product/alerts/alert-types/#metric-alerts) by following the same steps as above. Once you've chosen the metric you want to be alerted about, click "Add Action", then select Opsgenie from the notification types dropdown.
```

![Opsgenie metric alert rule action](opsgenie-metric-alert-rule.png)

> If a Opsgenie incident is created from the critical trigger of a metric alert, it will be acknowledged when the Sentry metric alert is resolved.

Reviewer: ```suggestion
Alerts that get resolved in Sentry will be automatically acknowledged in Opsgenie.
```
